examples/next-prisma-todomvc/src/pages/[filter].tsx

```diff
@@ -221,6 +221,7 @@ export default function TodosPage(props: PageProps) {
 
           <input
             className="new-todo"
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
             placeholder={t('what_needs_to_be_done') as string}
```

Comment:
This one stumped me. I couldn't reproduce it in a standalone repro: https://github.com/JoshuaKGoldberg/repros/tree/next-18n-and-type-assertion.

...are there some custom fancy types getting added? I time boxed investigating and couldn't find any.